packages/eslint-config-sentry-react/rules/imports.js

```diff
@@ -83,12 +78,11 @@ module.exports = {
 
     // Enforce a convention in module import order
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md
-    // TODO: enable?
     'import/order': [
-      'off',
+      'error',
       {
-        groups: [['builtin', 'external', 'internal'], 'parent', 'sibling', 'index'],
-        'newlines-between': 'ignore',
+        groups: ['builtin', 'external', 'internal', 'parent', 'sibling', 'index'],
```

Comment:
Let's group parent, sibling, and index together (we don't usually use parent)

```suggestion
        groups: ['builtin', 'external', 'internal', ['parent', 'sibling', 'index']],
```